down_frab_videos.py

```diff
@@ -241,6 +243,28 @@ def __init__(self, short_message, long_message):
         self.long_message = long_message
 
 
+def wrap_bs4(content):
```

Comment:
You could write this as a `for`-loop over a list `["lxml", "html.parser", "html5lib"]`. This makes it easer to extend the list at a later point.

Reply:
good point